app/src/pages/playground/Playground.tsx

```diff
@@ -254,35 +257,37 @@ function PlaygroundContent() {
       >
         <Panel>
           <div css={playgroundPromptPanelContentCSS}>
-            <Accordion arrowPosition="start" size="L">
-              <AccordionItem
-                title="Prompts"
-                id="prompts"
-                extra={
-                  <Flex direction="row" gap="size-100" alignItems="center">
-                    <TemplateLanguageRadioGroup />
-                    <AddPromptButton />
-                  </Flex>
-                }
-              >
-                <div css={promptsWrapCSS}>
-                  <Flex direction="row" gap="size-200" maxWidth="100%">
-                    {instances.map((instance) => (
-                      <View
-                        flex="1 1 0px"
-                        key={instance.id}
-                        minWidth={PLAYGROUND_PROMPT_PANEL_MIN_WIDTH}
-                      >
-                        <PlaygroundTemplate
+            <DisclosureGroup defaultExpandedKeys={["prompts"]}>
+              <Disclosure id="prompts">
+                <DisclosureTrigger arrowPosition="start">
+                  Prompts
+                  <StopPropagation>
+                    <Flex direction="row" gap="size-100" alignItems="center">
+                      <TemplateLanguageRadioGroup />
+                      <AddPromptButton />
+                    </Flex>
+                  </StopPropagation>
```

Comment:
There's a bit of visual regression here where I think the "actions" should still be right aligned

Also the addition of the nested values causes the trigger hight to grow. Can we use a different sizing strategy so that these triggers stay the same height? I know it looks odd padding wise but it might be the lesser of two evils right now?

Reply:
<img width="579" alt="image" src="https://github.com/user-attachments/assets/a3144ee6-10ed-4c7c-a45d-c76bb482fc51" />

I added a "L" variant that makes all of these disclosures the same height now